.github/workflows/test.yml

```diff
@@ -115,6 +114,8 @@ jobs:
       FORCE_COLOR: ${{ !(startsWith(matrix.os, 'windows-') && startsWith(matrix.toxenv, 'py')) && 1 || 0 }}
       # Tox
       PY_COLORS: 1
+      # Python -- Disable argparse help colors (3.14+)
+      PYTHON_COLORS: 0
```

Comment:
Not very familiar with the behavior of `PYTHON_COLORS`, what is the purpose of disabling them here?

Reply:
In #19021 I enabled color output for argparse help in Python 3.14. On linux we pass `FORCE_COLOR=1` as env variable thus enabling color during the CI run. `PYTHON_COLORS=0` overwrites that, but just for Python interpreter itself (i.e. we still get the colored pytest output). That way we won't have to deal with color escape codes in the tests. _There are a few which check the help output._

https://docs.python.org/3.14/using/cmdline.html#using-on-controlling-color

Reply:
Aha! So the motivating issue is escape codes in the tests, makes total sense.